compiler/qsc_project/src/manifest.rs

```diff
@@ -16,16 +16,20 @@ use std::path::PathBuf;
 pub const MANIFEST_FILE_NAME: &str = "qsharp.json";
 
 /// A Q# manifest, used to describe project metadata.
-#[derive(Deserialize, Debug, Default)]
+#[derive(Deserialize, Debug, Default, Clone)]
 pub struct Manifest {
     pub author: Option<String>,
     pub license: Option<String>,
-    #[serde(default)]
+    #[serde(default = "default_exclude_regexes")]
     pub exclude_regexes: Vec<String>,
     #[serde(default)]
     pub exclude_files: Vec<String>,
 }
 
+fn default_exclude_regexes() -> Vec<String> {
+    vec![".*node_modules.*".into(), ".*\\.git.*".into()]
```

Comment:
These shouldn't just be the default but they should be always added, IMO. Otherwise as soon as the user adds an exclude regex we'll start crawling .git again which isn't an ideal user experience.

Reply:
We decided to get rid of excludes (and RegExes), right?